.forge-snapshots/TickMathGetTickAtSqrtPrice.snap

```diff
@@ -1 +1 @@
-196153
\ No newline at end of file
+195023
```

Comment:
this is a different test it doesn tmake sense lol

Reply:
super odd.. I guess let's see if this becomes flaky? Not sure why this test changed this val